en/modules/cms-kit/poll.md

```diff
@@ -0,0 +1,124 @@
+# Poll System
+
+CMS kit provides a **poll** system to allow users to create, edit and delete polls. Here is a screenshot of the poll widget:
+
+![cmskit-module-poll-widget](../../images/cmskit-module-poll-widget.png)
+
+## User Interface
+
+### Menu Items
+
+**Polls**: Opens the poll management page.
+
+### Pages
+
+#### Polls
+
+You can list, create, update and delete on the admin side of your solution.
+
+![poll-page](../../images/cmskit-module-poll-page.png)
+![poll-edit-question-page](../../images/cmskit-module-poll-edit-question-page.png)
+![poll-edit-options-page](../../images/cmskit-module-poll-edit-options-page.png)
+
+## The Poll Widget
+
+The poll system provides a poll [widget](https://docs.abp.io/en/abp/latest/UI/AspNetCore/Widgets) for users to vote and show the result. You can place the widget on a page like the below:
+
+```csharp
+@await Component.InvokeAsync(
+    typeof(PollViewComponent),
+    new
+    {
+        widgetName = "poll"
```

Comment:
A name like `my-poll-1` would be better to understand this is a custom name. Also, mention that you can concurrently have multiple active poll widgets in your application with different widget names.